### PR TITLE
Move 1.8 tests to 2.0 tests 

### PR DESCRIPTION
**Pull request by GaoWei8**
- case

- cond

- sqrt
661caf156078fcde57d29b8b30d357e5

- floor 反向为None
98a107d1e9cc9aa330f3d604cf742d0a

- cos_sim
2.3接口返回为一个scale或与label的维度相同。都不能与paddle维度对齐。
https://www.tensorflow.org/api_docs/python/tf/compat/v1/losses/cosine_distance
e9e69bc2a29888b8f9939b07b99c112c

- dropout
random op，结果不能对齐

- while_loop 反向为None
0028c6314ccc661ee928cf4450be03b3

- split
7a0959218776e170c0e92897ef833408

- sigmoid

- slice

- softmax_with_cross_entropy.

### CODE REVIEW

File: api/tests_v2/cos_sim.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# limitations under the License.
+
+from common_import import *
+
+
+class PDCosSim(PaddleAPIBenchmarkBase):
+    def build_program(self, config):
+        x = self.variable(name='x', shape=config.X_shape, dtype=config.X_dtype)
+        y = self.variable(name='y', shape=config.Y_shape, dtype=config.Y_dtype)
+        result = paddle.fluid.layers.cos_sim(X=x, Y=y)
```
Comment (Xreki) [inline api/tests_v2/cos_sim.py:22]: 有2个接口：`paddle.metric.cos_sin`和`paddle.nn.functional.cosine_similarity(x1, x2, axis=1, eps=1e-08)`，都不能对齐吗？

**Review by Xreki — APPROVED** [review]
LGTM. cos_sim可以后面再看下
